samples/basic/blinky/src/main.c

```diff
@@ -25,14 +25,12 @@ int main(void)
 	int ret;
 	bool led_state = true;
 
-	if (!gpio_is_ready_dt(&led)) {
+	if (!gpio_is_ready_dt(&led))
 		return 0;
-	}
 
 	ret = gpio_pin_configure_dt(&led, GPIO_OUTPUT_ACTIVE);
-	if (ret < 0) {
+	if (ret < 0)
```

Comment:
was it supposed to catch this? This is not allowed

Reply:
this is confusing, I see that above.

Reply:
This was done per request of @kartben here: https://github.com/zephyrproject-rtos/zephyr/pull/75089#pullrequestreview-2148174067

I don't have a strong preference, as long as the notice is automated by the script I'm fine.

Reply:
@nashif as you can see the compliance check action actually succeeds, so the notations are merely suggestions, not blockers.

Reply:
> was it supposed to catch this? This is not allowed

This isn't allowed, but there currently isn't any compliance check that would prevent this.